src/compas_rhino/install.py

```diff
@@ -73,6 +68,14 @@ def install(version=None, packages=None):
 
     if not packages:
         packages = INSTALLABLE_PACKAGES
+        if system == 'darwin' and version == '5.0':
+            packages.remove('compas_ghpython')
+    elif system == 'darwin' and 'compas_ghpython' in packages:
```

Comment:
this seems to remove `compas_ghpython` from the packages list for both versions of Rhino for Mac. no?

Reply:
Yes. Should include ’and version == ”5.0”’. Will fix!